src/js/disability-benefits/components/ClaimsTimeline.jsx

```diff
@@ -16,7 +19,11 @@ export default class ClaimsTimeline extends React.Component {
           <p>Your claim has been assigned to a reviewer who is determining if additional information is needed.</p>
         </ClaimPhase>
         <ClaimPhase phase={3} current={userPhase} activity={activityByPhase} id={id}>
-          <p>If we need more information, we will request it from you, health care providers, governmental agencies, or others. Once we have all the information we need, we will review it and send your claim to the rating specialist for a decision.</p>
+          <p>
+            If we need more information, we will request it from you, health care providers, governmental agencies, or others. Once we have all the information we need, we will review it and send your claim to the rating specialist for a decision.
+            {everPhaseBack && ' There may be times when a claim moves forward to "Preparation for decision notification" and then temporarily back to this phase for further processing.'}
```

Comment:
@austin-martinez I'm confused what you mean by both phases. These changes only show up in phase 3 (there are two screenshots).